addons/website_sale/static/tests/tours/website_sale_mock_tracking.js

```diff
@@ -0,0 +1,45 @@
+odoo.define('website_sale.tour_mock_tracking', function (require) {
+
+const publicWidget = require('web.public.widget');
+const websiteSaleTracking = require('website_sale.tracking');
+
+require('website_sale.website_sale');
+
+const events = {
+    view_item: [],
+    add_to_cart: [],
+    remove_from_cart: [],
+};
+
+let promise;
+
+publicWidget.registry.WebsiteSale.include({
+    _getCombinationInfo() {
+        promise = this._super(...arguments);
+        return promise;
+    },
+});
+
+publicWidget.registry.WebsiteSale.getCombinationInfoPromise = function () {
+    let result = promise;
+    promise = undefined;
+    return result;
+};
+
+websiteSaleTracking.include({
+    _onViewItem(event, data) {
+        events.view_item.push(data);
+    },
+    _addToCart(event, data) {
```

Comment:
```suggestion
    _onAddToCart(event, data) {
```
:'( that's why the runbot was red and this test not working
took me a few time to figure....